packages/block-library/src/rss/edit.js

```diff
@@ -119,6 +120,8 @@ class RSSEdit extends Component {
 							onChange={ ( value ) => setAttributes( { itemsToShow: value } ) }
 							min={ DEFAULT_MIN_ITEMS }
 							max={ DEFAULT_MAX_ITEMS }
+							allowReset={ true }
+							initialValue={ this.state.initialItemsToShow }
```

Comment:
I noticed that there is already `initialPosition` prop which plays exactly the same role. We should use it instead. See:
https://github.com/WordPress/gutenberg/tree/c2c112846126e786939c2525a2c802d6016a2c82/packages/components/src/range-control#initialposition

Reply:
The initialPosistion prop updates each time the slider button is changed. If I simply reset to initialPosition, nothing would change. This is why I created the initialValue

Reply:
> The initialPosistion prop updates each time the slider button is changed. If I simply reset to initialPosition, nothing would change. This is why I created the initialValue

@jorgefilipecosta - I see that you have added `initialPosition` in https://github.com/WordPress/gutenberg/pull/6088. Can you clarify its intended behavior?

Reply:
The initialPosition property allows the slider circle to appear as if a given value was select, when in fact the value is undefined.
![image](https://user-images.githubusercontent.com/11271197/58055399-90020980-7b55-11e9-9219-b4b0ea801513.png)
This was the functionality that allowed the font size picker to appear in a position that makes sense instead of appearing in the middle when the font size is unset.
The font size picker was meanwhile redesigned and core is not using this feature anymore, but I guess we need to keep it in order to be back-compatible with existing plugins.

Reply:
Interesting, this means it's something different and its description should be updated to better reflect it. I remember what was the case, we wanted to show the current font size inherited from the CSS used by the theme author.